examples/source/waypoints/index.js

```diff
@@ -98,6 +98,8 @@ const WaypointsExample = ({
       <TourTracker
         tourers={decoratedTourers}
         selected={selected}
+        focusOnTourer={decoratedTourers[0].id}
```

Comment:
Why not use selected? Can you explain the behaviour this gives you?
Does it avoid unnecessary zooms on re-renders?
Do we think it'll get annoying?

Also could we follow the same sort of naming convention here: `focusedTourer`?

... in saying that it feels very much like `selected` fits that bill.